app/app.go

```diff
@@ -219,13 +236,18 @@ func New(
 		return nil, err
 	}
 
+	// Wire & initialize IBC module
+	app.initIBC()
+
 	// create the simulation manager and define the order of the modules for deterministic simulations
 	overrideModules := map[string]module.AppModuleSimulation{
 		authtypes.ModuleName: auth.NewAppModule(app.appCodec, app.AccountKeeper, authsims.RandomGenesisAccounts, app.GetSubspace(authtypes.ModuleName)),
 	}
 	app.sm = module.NewSimulationManagerFromAppModules(app.ModuleManager.Modules, overrideModules)
 	app.sm.RegisterStoreDecoders()
 
+	// TODO: Add UpgradeKeeper.SetUpgradeHandler() to migrate store for IBC support.
```

Comment:
👍🏻 

Yes, good call. You'll need to add store keys for both ibc core and the transfer module.
Probably need something like the following in your upgrade handler:

```go
upgradeInfo, err := app.UpgradeKeeper.ReadUpgradeInfoFromDisk()
if err != nil {
    panic(err)
}

if upgradeInfo.Name == UpgradeName && !app.UpgradeKeeper.IsSkipHeight(upgradeInfo.Height) {
	storeUpgrades := storetypes.StoreUpgrades{
		Added: []string{
			ibcexported.StoreKey,
			ibctransfertypes.StoreKey,
		},
		Deleted: []string{},
	}

	// configure store loader that checks if version == upgradeHeight and applies store upgrades
	app.SetStoreLoader(upgradetypes.UpgradeStoreLoader(upgradeInfo.Height, &storeUpgrades))
}
```